docs/deployment/pro.md

```diff
@@ -86,6 +86,8 @@ queueMode: jetstream
 
 nats:
   streamReplication: 1
+  authorization:
+    enabled: true
 ```
 
```

Comment:
Give the command for creating the nats-secret when upgrading an older/existing installation, or from OpenFaaS CE.